src/components/publications_list/Publication.tsx

```diff
@@ -0,0 +1,10 @@
+export interface Publication {
```

Comment:
Note: the entire `publications_list` component is basically a duplicate of the collaborators component with different fields. In the future it might make sense to deduplicate this component.